lib/observer/src/observer_sys_wx.erl

```diff
@@ -122,14 +131,19 @@ info_fields() ->
 	    ]}],
     Stats = [{"Statistics", right,
 	      [{"Up time", {time_ms, uptime}},
-	       {"Max Processes", process_limit},
-	       {"Processes", process_count},
 	       {"Run Queue", run_queue},
 	       {"IO Input",  {bytes, io_input}},
 	       {"IO Output", {bytes, io_output}}
 	      ]}
 	    ],
-    {Sys, Mem, Cpu, Stats}.
+	Limits = [{"System statistics / limit",
```

Comment:
The correct atoms names should be:
` atom_count_limit`, `process_count_limit` etc.. 

I left `*_limit` for backward compatibility